src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml

```diff
@@ -1,16 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!--Copyright, Microsoft Corporation, All rights reserved.-->
+<!--NO_TRANSLATE-->
 <Rule Name="ConfiguredBrowseObject"
-      Description="General"
-      DisplayName="General"
       OverrideMode="Replace"
       PageTemplate="generic"
       PropertyPagesHidden="true"
       xmlns="http://schemas.microsoft.com/build/2009/properties">
   <Rule.Categories>
-    <Category Name="General"
-              Description="General"
-              DisplayName="General" />
+    <Category Name="General" />
```

Comment:
If a rule doesn't show in the UI then it doesn't need categories at all, they are purely a UI concept.